common/src/main/java/whocraft/tardis_refined/common/capability/TardisLevelOperator.java

```diff
@@ -142,6 +142,8 @@ public void tick(ServerLevel level) {
             tardisClientData.setShellTheme(aestheticHandler.getShellTheme());
             tardisClientData.setShellPattern(aestheticHandler.shellPattern().id());
             tardisClientData.setHumEntry(interiorManager.getHumEntry());
+            tardisClientData.setFuel(pilotingManager.getFuel());
```

Comment:
I cannot wait for TardisClientData to die

Reply:
Agreed